app/components/global/layout/TwoColumnLayout.md

```diff
@@ -0,0 +1,10 @@
+Responsive layout with two columns on wider screens and a single column on
+smaller screens.
+
+```js
+;<TwoColumnLayout>
```

Comment:
Is the semi-colon necessary?

Reply:
I think in this specific case it isn't necessary as there is nothing before it. But if you were to add a JS statement between lines 4 and 5 it would become necessary.